modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -22,10 +24,17 @@ const (
 	// The controller chain channel version
 	flagVersion = "version"
 	// The channel ordering
-	flagOrdering              = "ordering"
-	flagRelativePacketTimeout = "relative-packet-timeout"
+	flagOrdering               = "ordering"
+	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
+	flagAbsoluteTimeouts       = "absolute-timeouts"
```

Comment:
what is this used for? Unless I am missing something the code never allows an absolute timeout to be used?

Reply:
we should either remove the absolute flag or allow absolute timestamps to be specified

Reply:
Isn't it the case that absolute timeout timestamp is used when `flagAbsoluteTimeouts` is true and `flagPacketTimeoutTimestamp` is also used?

Reply:
Isn't it the case that absolute timeout timestamp is used when `flagAbsoluteTimeouts` is true and `flagPacketTimeoutTimestamp` is also used?

Reply:
yes, (this logic was missing when I commented). In addition, the previous logic did not allow absolute timeout submission which is why I wasn't sure which direction was being chosen